docs/builder-guides/XVM and WASM/interact_with_wasm_contract.md

```diff
@@ -0,0 +1,318 @@
+---
+sidebar_position: 3
+---
+
+# Interact with WASM contract with astar.js
+## TL;DR
+During this guide, we'll be guiding you on how to interact with WASM contract with astar.js
```

Comment:
Guide twice in the same sentence looks a bit awkward to me.  "In this guide we'll see how…" or "Here we'll be guiding you…" IMO.